modules/manage/pages/console/edit-topic-configuration.adoc

```diff
@@ -1,8 +1,7 @@
-= Edit Topic Configuration in Redpanda Console
-:description: Use Redpanda Console to edit the configuration of existing topics in a cluster.
-:page-categories: Redpanda Console
+= Edit Topic Configuration in the Redpanda UI
```

Comment:
Now that we are going with the strategy of including all relevant content directly in cloud docs, we need to conditionalize this so that self-hosted users see 'Redpanda Console' and cloud users see 'Redpanda UI'.

I've added an attribute that we can use for this to https://github.com/redpanda-data/cloud-docs/pull/3

For example:

```asciidoc
ifdef::env-cloud[]
:ui: Redpanda Cloud
endif::[]
ifndef::env-cloud[]
:ui: Redpanda Console
endif::[]
= Edit Topic Configuration in {ui}
```

Reply:
@JakeSCahill and @micheleRP. {ui} attribute added.